### PR TITLE
Add docs for keys modification

### PR DESCRIPTION
**Pull request by ppot**

### CODE REVIEW

File: app/config/config-default.js [hunk 1]
```diff
@@ -116,5 +116,10 @@ module.exports = {
   // in development, you can create a directory under
   // `~/.hyper_plugins/local/` and include it here
   // to load it and avoid it being `npm install`ed
-  localPlugins: []
+  localPlugins: [],
+
+  keymaps: {
+    // exemple
```
Comment (CodeTheory) [inline app/config/config-default.js]: Example*

File: website/index.html [hunk 2]
```diff
@@ -1,1244 +1,1200 @@
...
...zs/Q0dLT1NXW19jZ2tvc3d7f4OHi4+Tl5ufo6err7O3u7/Dx8vP09fb3+Pn6+/z9/v8AA36zIUMgrgIGcdlIyBCdgUOBAAAh+QQFBAB/ACwAAAAAAQABAAAHA4B/gQAh+QQFBAB/ACw8AEAAGwAGAAAHFIB+goOEhYaHiImKi4yNjo+QkYmBACH5BAUEAH8ALAIACQC6AHcAAAf/gH+Cg39fCYSIiYqLjI2Oj5CRkpOUlZaXmJmam5ydlX6eoaKjpKWmp6ipqqusra6vsLGys7S1tre4ubq7vL2+v8DBwsPExcbHyMnKy8zNzsF6z9LT1NXW15c42Nvc3d7f4OHi4+Tl5ufeB+jkKuvu7/Dx8vP09fb3+NMm+fz9uwT+AgocSLCgwYMIEypcyLChw4cQI0qcSLGixYsYM2rcyLHjxAIkPMKyIRJWgZIoU6pcybKly5cwY8qcSbOmzZs4c+rcybOnz59AgwodSrSo0aNIkypdyrSp06dQo0qdSrWq1atYs2rdyrWr169gw4o95mPszw4dopYV1IHC1LRKAgMBACH5BAUEAH8ALAUAFQC3AGsAAAfggH+Cg4SFhoeIiYqLjI2Oj5CRkoVtk5aXmJmam5ydnp+goaKjpKWmp6ipqqusra6vsLGys7S1tre4ubq7vL2+v8DBwsPExcbHyMnKy8zNzs/Q0dLT1NXW19jZh27a3d7f4OHi4+Tl5ufo6err7O3u7/Dx8vP09fb3+Pn6+/z9/v8AAwosZUPGQFwFDuKyobChw4cQI0qcSLGixYsYM2rcyLGjx48gQ4ocSbKkyZMoU6pcybKly5cwY8qcSbOmzZs4c+rcybOnz59AgwodSrSo0aNIkypdyrSpxQkShk7wGQgAIfkEBQQAfwAsAAAAAAEAAQAABwOAf4EAIfkEBQQAfwAsAAAAAAEAAQAABwOAf4EAIfkEBQQAfwAsBQAaALMAZgAAB9GAf4KDhIWGh4iJiouMjY6PkJGShzCTlpeYmZqbnJ2en6ChoqOkpaanqKmqq6ytrq+wsbKztLW2t7i5uru8vb6/wMHCw8TFxsfIycrLzM3Oz9DR0tPU1dbX2Nna29zd3t/g4eLj5OXm5+jp6uvs7e7v8PHy8/T19vf4+frFC/v+/wADChxIsKDBgwgTKlzIsKHDhxAjSpxIsaLFixgzatzIsaPHjyBDihxJsqTJ
```
Comment (chabou) [inline website/index.html]: It is an object not an array

Comment (chabou) [inline website/index.html]: Maybe a link on "All command keys" ?

Comment (ppot) [inline website/index.html]: 🤔 This is already here.

File: app/config/config-default.js [hunk 3]
```diff
@@ -116,5 +116,10 @@ module.exports = {
   // in development, you can create a directory under
   // `~/.hyper_plugins/local/` and include it here
   // to load it and avoid it being `npm install`ed
-  localPlugins: []
+  localPlugins: [],
+
+  keymaps: {
+    // Exemple
```
Comment (CodeTheory) [inline app/config/config-default.js]: `Example`*

File: website/index.html [hunk 4]
```diff
@@ -677,6 +677,23 @@ <h2 id="extensions"><a href="#extensions">Extensions</a></h2>
...
+    
+          <P> All command keys can be changed. In order to change them, edit 
+             <code>~/.hyper.js</code> and add your desired change to <code>keymaps</code>.
+           </p>
+           <p> Then Hyper will change the default with your custom change.</p>
+           <p> Example: <code>'window:devtools': 'Cmd+Alt+O'</code> <p>
+             <pre><code>module.exports = {
+  config: { /*... */ },
+
+  keymaps: [
```
Comment (chabou) [inline website/index.html]: kaymaps is an object not an array

**Comment by ppot** [comment]
Changed

File: app/config/config-default.js (hunk 3 above)
Comment (ppot) [inline app/config/config-default.js]: Done

File: website/index.html (hunk 4 above)
Comment (ppot) [inline website/index.html]: Done

File: app/config/config-default.js (hunk 1 above)
Comment (ppot) [inline app/config/config-default.js]: Done

File: website/index.html [hunk 5]
```diff
@@ -677,6 +677,23 @@ <h2 id="extensions"><a href="#extensions">Extensions</a></h2>
         <p>You can also take a look at
           <a href="https://github.com/bnb/awesome-hyper">Awesome Hyper</a>
           for a curated list of plugins and resources.</p>
+          
+          <h2 id="keymaps"><a href="#keymaps">Keymaps</a></h2>
+    
+          <P> All command keys can be changed. In order to change them, edit
```
Comment (chabou) [inline website/index.html]: Maybe add a link to default keymap files on 'All command keys' to list all available commands?

**Comment by chabou** [comment]
Approved to not block merge. But I really think that we need a list of commands

File: website/index.html (hunk 5 above)
Comment (rafegoldberg) [inline website/index.html]: some way to view all default keymappings would be much appreciated!

**Pull request merged**